src/pages/MovieDetail/MovieDetail.js

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import MainInfo from './components/MainInfo';
+import InfoPage from './components/InfoPage';
+import CommentList from './components/CommentList';
+import Trailer from './components/Trailer';
 
 const MovieDetail = () => {
-  return <div>MovieDetail</div>;
+  const [movieInfo, setMovieInfo] = useState('');
```

Comment:
들어올 데이터가 객체이기 때문에, 어떤 형태의 데이터인지 알 수 있게 초기값으로 빈 객체를 넣어주는 게 좋을 것 같습니다!
```suggestion
  const [movieInfo, setMovieInfo] = useState({});
```

Reply:
말씀하여 주신 부분, 빈 객체로 지정하니 movieInfo의 초기값이 undefiend으로 설정되어 movieInfo의 키값에 붙는 toString()을 못 읽어 오류가 생기는데, 빈객체로 지정 후 어떻게 해결해야 할까요?\

Reply:
기존 movieInfo의 초기값은 `undefined`가 아닌 빈 string(`''`)이었기 때문에, 아래에서 JSX return할 때 조건부 렌더링으로 구현해 동작하고 있었습니다(빈 string은 falsy한 값이기 때문).
이때 초기값을 빈 객체(`undefined` X)로 부여하게 되면 truthy한 값이기 때문에 아래의 조건부 렌더링이 불필요해진다는 설명이었습니다.
그렇기 때문에 실제 데이터가 들어온 경우에 조건부 렌더링으로 화면을 다르게 보여주시면 됩니다!
(초기의 movieInfo는 빈 객체이고, 실제 데이터의 key에 접근하려고 하면 `undefined`가 뜨는 게 정상입니다)